lib/Serialization/ModuleFile.cpp

```diff
@@ -388,8 +388,7 @@ class ModuleFile::DeclTableInfo {
 
   hash_value_type ComputeHash(internal_key_type key) {
     if (key.first == DeclBaseName::Kind::Normal) {
-      // FIXME: DJB seed=0, audit whether the default seed could be used.
-      return llvm::djbHash(key.second, 0);
+      return llvm::djbHash(key.second, SWIFTMODULE_HASH_SEED);
```

Comment:
Perhaps factor this out into a ModuleFile::hashFunction(). I'm not sure whether this is much better, I'll leave it up to you.

Reply:
Yeah, I considered that, but figured this was slightly more likely to keep people from doing the wrong thing in the future. I'm not 100% sure of that though.